python-libraries/nanover-openmm/src/nanover/openmm/cli.py

```diff
@@ -63,6 +63,20 @@ def handle_user_arguments() -> argparse.Namespace:
         metavar="STEPS",
         help="Update the interactions every STEPS dynamics steps.",
     )
+    parser.add_argument(
+        "-q",
+        "--include-velocities",
+        type=bool,
+        default=False,
+        help="Optionally include the particle velocities in the frame data.",
+    )
+    parser.add_argument(
+        "-k",
+        "--include-forces",
+        type=bool,
+        default=False,
+        help="Optionally include the particle forces in the frame data.",
+    )
```

Comment:
These should be `action="store_true"` with no `type=bool` -- otherwise it expects a value to be provided

Reply:
Added this change, thank you for spotting this!